README.md

```diff
@@ -1,107 +1,177 @@
-did-io
-=================
+# DID Client _(did-io)_
```

Comment:
We do also have the "did-client" project.  Should this just say "DID IO"?

Reply:
How about we rename `did-client` repo to `did-cli`?

Reply:
File an issue in did-client about this please.

Reply:
Done.